datasets/nrcan-landcover/nrcan-landcover-example.ipynb

```diff
@@ -0,0 +1,1352 @@
+       "| 0.00,-30.00, 586590.00|\n",
+       "| 0.00, 0.00, 1.00|</dd><dt><span>resolution :</span></dt><dd>30.0</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray 'stackstac-94f5f71e7b85a6b5b00d5c297a8b05a1' (y: 6002, x: 6855)>\n",
+       "array([[18., 18., 18., ...,  1.,  1.,  1.],\n",
+       "       [18., 18., 18., ...,  1.,  1.,  1.],\n",
+       "       [18., 18., 18., ...,  1.,  1.,  1.],\n",
+       "       ...,\n",
+       "       [ 1.,  1.,  1., ...,  1.,  1.,  1.],\n",
+       "       [ 6.,  6.,  6., ...,  1.,  1.,  1.],\n",
+       "       [ 6.,  6.,  6., ...,  1.,  1.,  1.]])\n",
+       "Coordinates: (12/19)\n",
+       "    band               <U9 'landcover'\n",
+       "  * x                  (x) float64 -2.058e+06 -2.058e+06 ... -1.852e+06\n",
+       "  * y                  (y) float64 5.866e+05 5.866e+05 ... 4.066e+05 4.066e+05\n",
+       "    proj:epsg          int64 3978\n",
+       "    proj:wkt2          <U1345 'PROJCRS[\"NAD83 / Canada Atlas Lambert\",BASEGEO...\n",
+       "    label:classes      object {'name': '', 'classes': ['Temperate or sub-pola...\n",
+       "    ...                 ...\n",
+       "    title              <U24 'Land cover of Canada COG'\n",
+       "    label:type         <U6 'raster'\n",
+       "    end_datetime       <U20 '2020-01-01T00:00:00Z'\n",
+       "    sci:doi            <U14 '10.4095/315659'\n",
+       "    raster:bands       object {'nodata': 0, 'sampling': 'area', 'data_type': ...\n",
+       "    epsg               int64 3978\n",
+       "Attributes:\n",
+       "    spec:        RasterSpec(epsg=3978, bounds=(-2057730.0, 406530.0, -1852080...\n",
+       "    crs:         epsg:3978\n",
+       "    transform:   | 30.00, 0.00,-2057730.00|\\n| 0.00,-30.00, 586590.00|\\n| 0.0...\n",
+       "    resolution:  30.0"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "merged = stackstac.mosaic(stack, dim=\"time\", axis=None).squeeze().compute()\n",
```

Comment:
I worry that this isn't doing what we hope. IIUC, `stackstac.mosiac` uses just `np.nan` as a sentinel value. But I think we're using 0 (or whatever is in the `raster` extension). So I think we might incorrectly pick up a 0 value here (maybe it doesn't matter if there's not any overlap?).

Reply:
You're right. The current usage was removing the overlapping area between two images because I wasn't using `np.nan` as the nodata value, which `stackstac.mosaic` assumes.